src/components/Banner.jsx

```diff
@@ -7,76 +7,75 @@ class Banner extends React.Component {
 
   renderLogos = allLogos => (
     <Card fluid >
-      <Card.Content className="card">
-        {this.getRandom(allLogos, 4).map(logo => {
-        if(logo.node.image.file.url != null){
+      <Card.Content className="cardcontent">
+        {getRandom(allLogos, 5).map(logo => {
           return(
             <Image className="image" key={logo.node.id} size="small" src={logo.node.image.file.url} />
           );
-        }
-        return null;
-      })}
+        })}
       </Card.Content>
     </Card>
   )
 
-  getRandom = (arr, n) => {
-    var result = new Array(n),
-    len = arr.length,
-    taken = new Array(len);
-
-    if(n > len)
-      return arr;
-
-    while(n--){
-      var x = Math.floor(Math.random() * len);
-      result[n] = arr[x in taken ? taken[x] : x];
-      taken[x] = --len in taken ? taken[len] : len;
-    }
-    return result;
-  }
-
   render(){
     return(
-      <BannerWrapper>
+      <BannerWrapper className="banner">
         {this.renderLogos(this.props.data.allContentfulBannerLogo.edges)}
         </BannerWrapper>
       );
   }
 }
 
+const getRandom = (arr, n) => {
+  const result = new Array(n);
+  let len = arr.length;
+  const taken = new Array(len);
+
+  if(n > len)
+    return arr;
+
+  while(n--){
+    const x = Math.floor(Math.random() * len);
+    result[n] = arr[x in taken ? taken[x] : x];
+    taken[x] = --len in taken ? taken[len] : len;
+  }
+  return result;
+}
+
 const BannerWrapper = styled.div`
   padding-bottom: 3em;
-  .card{
+  .cardcontent{
```

Comment:
I like the name change, but please use the snake-case convention for css classes